descartes/package.xml

```diff
@@ -16,7 +16,7 @@
   <run_depend>descartes_planner</run_depend>
   <run_depend>descartes_trajectory</run_depend>
   <run_depend>descartes_utilities</run_depend>
-
+  <run_depend>descartes_tests</run_depend>
```

Comment:
`descartes` is a meta-package, correct?  Should we include a test package in the dependency list?  This will include it in the debian, even though none of the other packages really need it.  Mostly, curious.